[scheduled-executor] ClientScheduledExecutorServiceBasicTest.getErroneous_durable

```
Expected: (an instance of java.util.concurrent.ExecutionException and exception with cause expects type <class java.lang.IllegalStateException> with message "Erroneous task")
     but: exception with cause expects type <class java.lang.IllegalStateException> with message "Erroneous task" cause was <java.io.IOException: Partition does not have an owner. partitionId: 74>
Stacktrace was: java.util.concurrent.ExecutionException: com.hazelcast.client.HazelcastClientNotActiveException: Partition does not have an owner. partitionId: 74
 at com.hazelcast.client.spi.impl.ClientInvocationFuture.resolveAndThrowIfException(ClientInvocationFuture.java:109)
 at com.hazelcast.client.spi.impl.ClientInvocationFuture.resolveAndThrowIfException(ClientInvocationFuture.java:33)
 at com.hazelcast.spi.impl.AbstractInvocationFuture.get(AbstractInvocationFuture.java:183)
 at com.hazelcast.client.proxy.ClientScheduledFutureProxy.get0(ClientScheduledFutureProxy.java:225)
 at com.hazelcast.client.proxy.ClientScheduledFutureProxy.get(ClientScheduledFutureProxy.java:235)
 at com.hazelcast.scheduledexecutor.ScheduledExecutorServiceBasicTest.getErroneous_durable(ScheduledExecutorServiceBasicTest.java:1065)
 at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
 at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
 at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
 at java.lang.reflect.Method.invoke(Method.java:622)
 at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
 at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
 at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
 at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
 at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:105)
 at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:97)
 at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
 at java.util.concurrent.FutureTask.run(FutureTask.java:166)
 at java.lang.Thread.run(Thread.java:704)
Caused by: com.hazelcast.client.HazelcastClientNotActiveException: Partition does not have an owner. partitionId: 74
 at com.hazelcast.client.spi.impl.ClientInvocation.notifyException(ClientInvocation.java:189)
 at com.hazelcast.client.spi.impl.ClientInvocation.invokeOnSelection(ClientInvocation.java:156)
 at com.hazelcast.client.spi.impl.ClientInvocation.retry(ClientInvocation.java:174)
 at com.hazelcast.client.spi.impl.ClientInvocation.run(ClientInvocation.java:162)
 at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
 at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
 at java.util.concurrent.FutureTask.run(FutureTask.java:166)
 at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$101(ScheduledThreadPoolExecutor.java:165)
 at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:266)
 at com.hazelcast.util.executor.LoggingScheduledExecutor$LoggingDelegatingFuture.run(LoggingScheduledExecutor.java:140)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
 at java.lang.Thread.run(Thread.java:704)
 at com.hazelcast.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:64)
 at com.hazelcast.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:80)
 at ------ submitted from ------.(Unknown Source)
 at com.hazelcast.client.spi.impl.ClientInvocationFuture.resolveAndThrowIfException(ClientInvocationFuture.java:96)
 ... 18 more
Caused by: java.io.IOException: Partition does not have an owner. partitionId: 74
 at com.hazelcast.client.spi.impl.ClientSmartInvocationServiceImpl.invokeOnPartitionOwner(ClientSmartInvocationServiceImpl.java:42)
 at com.hazelcast.client.spi.impl.ClientInvocation.invokeOnSelection(ClientInvocation.java:146)
 at com.hazelcast.client.spi.impl.ClientInvocation.retry(ClientInvocation.java:174)
 at com.hazelcast.client.spi.impl.ClientInvocation.run(ClientInvocation.java:162)
 at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
 at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:334)
 at java.util.concurrent.FutureTask.run(FutureTask.java:166)
 at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$101(ScheduledThreadPoolExecutor.java:165)
 at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:266)
 at com.hazelcast.util.executor.LoggingScheduledExecutor$LoggingDelegatingFuture.run(LoggingScheduledExecutor.java:140)
 at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1146)
 at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
 at java.lang.Thread.run(Thread.java:704)
 at com.hazelcast.util.executor.HazelcastManagedThread.executeRun(HazelcastManagedThread.java:64)
 at com.hazelcast.util.executor.HazelcastManagedThread.run(HazelcastManagedThread.java:80)
```

https://hazelcast-l337.ci.cloudbees.com/view/Hazelcast/job/Hazelcast-3.x-ZuluJDK6-freestyle/229/testReport/junit/com.hazelcast.client.scheduledexecutor/ClientScheduledExecutorServiceBasicTest/getErroneous_durable/

--- a/hazelcast-client/src/test/java/com/hazelcast/client/scheduledexecutor/ClientScheduledExecutorServiceBasicTest.java
+++ b/hazelcast-client/src/test/java/com/hazelcast/client/scheduledexecutor/ClientScheduledExecutorServiceBasicTest.java
@@ -16,6 +16,7 @@
 
 package com.hazelcast.client.scheduledexecutor;
 
+import com.hazelcast.client.config.ClientConfig;
 import com.hazelcast.client.test.TestHazelcastFactory;
 import com.hazelcast.config.Config;
 import com.hazelcast.core.HazelcastInstance;
@@ -60,7 +61,9 @@ public class ClientScheduledExecutorServiceBasicTest extends ScheduledExecutorSe
 
     @Override
     public IScheduledExecutorService getScheduledExecutor(HazelcastInstance[] instances, String name) {
-        return factory.newHazelcastClient().getScheduledExecutorService(name);
+        ClientConfig config = new ClientConfig();
+        config.getNetworkConfig().setConnectionAttemptLimit(Integer.MAX_VALUE);
+        return factory.newHazelcastClient(config).getScheduledExecutorService(name);
     }
 
     @Override

